src/components/VariantSearch.tsx

```diff
@@ -4,18 +4,34 @@ import React, { useState } from 'react';
 import { VariantSearchField } from './VariantSearchField';
 import { AnalysisMode } from '../data/AnalysisMode';
 
+type SearchType = 'aa-mutation' | 'nuc-mutation' | 'pango-lineage';
+
+type SearchOption = {
+  label: string;
+  value: string;
+  type: SearchType;
+};
+
```

Comment:
Please avoid redundant type definitions! The same definition is in `VariantSearchField`.